### DISCUSSION TITLE
No top navigation to get back to the Projects Repos

### DISCUSSION THREAD

**Discussion by tralawar** [community/community#10088] [Projects and Issues]
I find it difficult to navigate to back to a Repo's overview, issues, or pull requests, etc.. from the Project board (beta)

**Comment by krzysztofrewak** [comment]
True, it's super annoying.

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448

**Comment by ucheNkadiCode** [comment]
Yes it would be helpful if the projects beta had a "Back to Repo" button

**Comment by jhodapp** [comment]
Agreed, it's very annoying. Navigation and UX is truly one of the most critical things the more functionality GitHub integrates into its offering and this is a prime example where it falls down flat and hits me every day.
